### PR TITLE
[CDAP-12744][CDAP-12746] dropdown for s3 regions and show last modified as date

### PR DESCRIPTION
**Pull request by elfenheart**
JIRAs:
- https://issues.cask.co/browse/CDAP-12744
- https://issues.cask.co/browse/CDAP-12746

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/S3Browser/BucketData/index.js [hunk 1]
```diff
@@ -142,29 +143,33 @@ const TableContents = ({enableRouting, search, data, onWorkspaceCreate, prefix,
...
-                <div className="col-xs-3">
-                  {file['owner']}
-                </div>
-                <div className="col-xs-3">
-                  {file['size']}
-                </div>
-                <div className="col-xs-3">
-                  {file['last-modified']}
+          filteredData.map(file => {
+            let lastModified = moment(file['last-modified']).format('MM/DD/YY HH:mm');
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/S3Browser/BucketData/index.js]: Should we probably use the `humanReadableDate` from the helper for consistency?

**Review by ajainarayanan — COMMENTED** [review]
Just a minor comment.

**Review by ajainarayanan — APPROVED** [review]
Code 👍 LGTM

**Comment by leacd** [comment]
LGTM

**Pull request merged**